package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "ourcord",
   "version": "1.1.4",
-  "main": "dist/websocket",
+  "main": "dist/websocket.js",
   "repository": "https://github.com/ourcord/ourcord",
   "author": "ourcord",
   "license": "MIT",
-  "types": "src/websocket.d.ts",
+  "types": "dist/websocket.d.ts",
+  "runkitExampleFilename": "./examples/example.js",
   "scripts": {
-    "start": "node dist/example.index.js",
-    "upgrade": "node packager.js",
-    "prepublish": "tsc"
+    "prepare": "npm run compile",
```

Comment:
What does this script and the 2 below this one do?

Reply:
Basiclly run script on spefic os,
We usually clean the output file before we compile it.

Reply:
Although it may be better i dont really see the point of this, we compile the src into dist before publishing it to npm

Reply:
Can we talk on discord dm

Reply:
Sure